pyproject.toml

```diff
@@ -90,18 +90,26 @@ venvPath = "."
 venv = ".venv"
 strict = ["src/mcp/**/*.py"]
 
-[tool.ruff.lint]
-select = ["C4", "E", "F", "I", "PERF", "UP"]
-ignore = ["PERF203"]
-
 [tool.ruff]
 line-length = 88
 target-version = "py310"
 
+[tool.ruff.lint]
+select = ["C4", "C90", "E", "F", "I", "PERF", "PL", "UP"]
```

Comment:
I think we should have each rule in a single line and have the link as a comment inline.

Reply:
This is unrelated to this PR.